pgctl/poll_ready.py

```diff
@@ -10,12 +10,16 @@
 
 import os
 import os.path
+import select
 import time
 
 from .functions import exec_
 from .functions import print_stderr
 
 
+DOWN_FIFO_PATH = os.path.join('event', 'ftrig1' + 'poll_ready'.ljust(43, '_'))
```

Comment:
Should add a comment, referencing https://github.com/skarnet/s6/blob/v2.2.2.0/src/libs6/ftrigw_notifyb_nosig.c#L29,L30